### DISCUSSION TITLE
凭据中的`1Remote\SecondaryVerificationEnabled`是什么？会不会影响配置的便携性？

### DISCUSSION THREAD

**Discussion by mo-han** [1Remote/1Remote#679] [Q&A]
1Remote\SecondaryVerificationEnabled 的作用是什么

**Comment by VShawn** [comment]
设置 SecondaryVerificationEnabled 凭据后，启动连接前将会要求验证Windows hello 人脸识别、指纹识别、pin 或密码。

该配置不影响任何部署相关功能，即便设置了 SecondaryVerificationEnabled，直接把exe和数据库拷到其他机器，也可以直接运行并开启远程。
